### ISSUE TITLE
AMD compatible distribution of jquery ui

### ISSUE THREAD

**Issue by koyadume** [jquery/learn.jquery.com#664]
[This](https://learn.jquery.com/jquery-ui/environments/amd/) page describes how one can use jquery ui with AMD. However when I download jquery ui distribution, its having all jquery ui plugins in a single js file instead of having one js file per plugin as described in the article. When can I download AMD compatible distribution of jquery ui?

**Comment by arthurvr** [comment]
I agree that there are definitely some ways this page can improve. Mainly this paragraph:

>  If you don't have a version of jQuery core handy, you can get it from http://jquery.com/download/, and you can download jQuery UI directly from http://jqueryui.com/. Alternatively you can download these libraries using a package manager such as Bower.

should be a little more specific how and where to download the packages. Thanks for reporting!
